main_test.go

```diff
@@ -240,59 +240,91 @@ var _ = Describe("github-review-helper", func() {
 				})
 
 				Context("with a +1 comment", func() {
-					BeforeEach(func() {
-						requestJSON = `{
-  "issue": {
-    "number": 7,
-    "pull_request": {
-      "url": "https://api.github.com/repos/salemove/github-review-helper/pulls/7"
-    }
-  },
-  "comment": {
-    "body": "+1, awesome job!"
-  },
-  "repository": {
-    "name": "github-review-helper",
-    "owner": {
-      "login": "salemove"
-    },
-    "ssh_url": "git@github.com:salemove/github-review-helper.git"
-  }
-}`
-						mockSignature()
-					})
+					var itMarksCommitPeerReviewed = func() {
+						Context("with GitHub request failing", func() {
+							BeforeEach(func() {
+								pullRequests.On("Get", "salemove", "github-review-helper", 7).Return(nil, nil, errors.New("an error"))
+							})
 
-					Context("with GitHub request failing", func() {
-						BeforeEach(func() {
-							pullRequests.On("Get", "salemove", "github-review-helper", 7).Return(nil, nil, errors.New("an error"))
+							It("fails with a gateway error", func() {
+								handle()
+								Expect(responseRecorder.Code).To(Equal(http.StatusBadGateway))
+							})
 						})
 
-						It("fails with a gateway error", func() {
-							handle()
-							Expect(responseRecorder.Code).To(Equal(http.StatusBadGateway))
+						Context("with GitHub request succeeding", func() {
+							BeforeEach(func() {
+								pullRequests.On("Get", "salemove", "github-review-helper", 7).Return(&github.PullRequest{
```

Comment:
[Already done](https://github.com/salemove/github-review-helper/commit/c6bbac2f9c9ba76929dc7c9d33b31297553f9347) as a part of the work for adding the !merge command.